content/en/serverless/guide/layer_not_authorized.md

```diff
@@ -0,0 +1,38 @@
+---
+title: Troubleshooting Serverless Layer not Authorized Errors
+kind: documentation
+---
+This guide helps you troubleshoot the deployment error `not authorized to perform: lambda:GetLayerVersion on resource`. This error is commonly seen with Datadog Library Layers or the Datadog Extension layer.
+
+## Regionality
+Lambda functions can only include [Lambda Layers][1] which are located in the same region as the function. Typically this error occurs when users copy instrumentation settings from other applications deployed in different regions.
+
+Verify that the Lambda Layer region and Lambda function version match. Then verify that the version number is correct.
+
+You can verify that a Lambda Layer version exists using any valid AWS credentials and running the following command:
+```
+aws lambda get-layer-version \
+  --layer-name arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Node16-x \
+  --version-number {{< latest-lambda-layer-version layer="node" >}}
+
+aws lambda get-layer-version \
+  --layer-name arn:aws:lambda:us-east-1:464622532012:layer:Datadog-Extension \
+  --version-number {{< latest-lambda-layer-version layer="extension" >}}
+```
```

Comment:
The example below from the package too large doc includes

> Replace the layer name and version number in the following example with those used by your applications:

I'm thinking we should add that line here too.

Reply:
My goal here is to list a couple of examples, I think that's okay because the error message will give them the exact layer name and version, so they can plug that into the command.

Happy to hear feedback on specific wording if you think that's confusing though. Thanks!

Reply:
We don't need to list a command for every layer, but we need to be explicit that the command we give here is just an example. I'll make a suggestion to the wording above.